lib/cmd/common/undeploy.js

```diff
@@ -10,34 +10,35 @@ var fhreq = require("../../utils/request");
 var common = require("../../common");
 var ini = require('../../utils/ini');
 
-// Main undeploy entry point
+var DELETEAPP_ENDPOINT = "api/v2/mbaas/deleteapp";
+
+/**
+ * Main undeploy entry point
+ * required args are id, env and domain
+ * `embed` can be specified and will be sent in the request to supercore
+ */
 function undeploy(argv, cb) {
-  var args = argv._;
-  if (args.length === 0) {
+  if (!argv.id || !argv.env || !argv.domain) {
     return cb(undeploy.usage);
   }
 
-  var appId = fhc.appId(args[0]);
-  var appType = args[1];
   var deployTarget = ini.getEnvironment(argv);
-
-  return undeployApp(appId, deployTarget, appType, cb);
+  return undeployApp(argv.id, deployTarget, argv.domain, argv.embed, cb);
 }
 
-function undeployApp(appId, deployTarget, appType, cb) {
-  var type = appType || "cloud";
-  var params = {
-    guid: appId,
-    apptype: type,
-    deploytarget: deployTarget
+function undeployApp(appId, deployTarget, domain, embed, cb) {
+  var body = {
+    embed: embed || false
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1125)